Site/css/style.css

```diff
@@ -445,6 +427,10 @@ form textarea {
 	font-size: 125%;
 }
 
+.content_container, .container{
```

Comment:
Skal kanskje dette referere til ID i stedet?

Reply:
Inga driver med CSS, så prøver å ikke rote for mye med den. 
Dette var lagt til for å prøve få footer litt vekk fra annet innhold, og fant forskjellige class på de ulike HTML sidene

Reply:
yeah, men jeg fant ikke noe html-element med content_container-klassen